tests/blob/test_blob_spec.lua

```diff
@@ -1,7 +1,7 @@
 -- Tests for luagit2's Blob module functions
 
 local fixer = require("Fixtures.fix_repo")
-local lfs = require("lfs")
+
```

Comment:
why is this removed here?

Reply:
There was no use of lfs module to check the path in this test. Therefore I removed it.

Reply:
ok!